### PR TITLE
Crpl 1966 add missing prototypes for web

### PR DESCRIPTION
**Pull request by csabee1979**

### CODE REVIEW

File: io/http/html post request dispatcher/88ec6117-e970-47ca-81f6-70c633880e85.json [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+        "type": "{string: string or string[]}",
+        "name": "form data",
+        "description": "Receives the form data to be posted.\n\nExample:\n{\n  \"custname\": \"John Doe\",\n  \"custtel\": \"1234567890\", \n  \"custemail\": \"john.doe@email.doe\",  \n\"size\": \"small\",\n\"topping\": [\"bacon\", \"cheese\", \"onion\"]\n}"
+      }
+    },
+    "outputs": {
+      "e58ea4f2-82c1-41c1-83f2-0a564421c408": {
+        "type": "number",
+        "name": "status",
+        "description": "Sends http response status code. Indicates whether the request has been  successfully completed.\n\nExample:\n200"
```
Comment (lvass74) [inline io/http/html post request dispatcher/88ec6117-e970-47ca-81f6-70c633880e85.json:25]: I think HTTP should be written in all caps the same way as HTML

File: io/http/html post request dispatcher/88ec6117-e970-47ca-81f6-70c633880e85.json [hunk 2]
```diff
@@ -0,0 +1,172 @@
+{
+  "name": "io/http/HTML post request dispatcher",
```
Comment (lvass74) [inline io/http/html post request dispatcher/88ec6117-e970-47ca-81f6-70c633880e85.json:2]: I'm a bit hesitant regarding the name, especially the term "HTML post" as technically HTML does not define anything like "post".
Should it rather be "HTML form post" as in the description?

**Review by lvass74 — CHANGES_REQUESTED** [review]

File: io/http/html post request dispatcher/88ec6117-e970-47ca-81f6-70c633880e85.json (hunk 2 above)
Comment (danstocker) [reply io/http/html post request dispatcher/88ec6117-e970-47ca-81f6-70c633880e85.json:2]: I'd also put HTTP methods in all-caps as a convention (POST, GET, etc.)

File: io/http/html form post request dispatcher/88ec6117-e970-47ca-81f6-70c633880e85.json [hunk 3]
```diff
@@ -0,0 +1,173 @@
+{
+  "name": "io/http/HTML Form POST request dispatcher",
+  "interface": {
+    "inputs": {
+      "3a16460c-b25e-40ca-a468-e15baf34c40d": {
+        "type": "string",
+        "name": "URL",
+        "description": "Receives the target of the HTML post.\n\nExample:\n\"https://httpbin.org/post\""
```
Comment (lvass74) [inline io/http/html form post request dispatcher/88ec6117-e970-47ca-81f6-70c633880e85.json:8]: "Receives the target of the HTML **form** post"

**Review by lvass74 — APPROVED** [review]

**Pull request merged**